### PR TITLE
Issue 102

### PR DESCRIPTION
**Pull request by sk8higher**

### CODE REVIEW

File: spec/models/browse_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (XaoGao) [inline spec/models/browse_spec.rb:1]: @sk8higher double quote

File: spec/factories/browses.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :browse do
+    user { nil }
```
Comment (XaoGao) [inline spec/factories/browses.rb:3]: why nil, fix to normal entities

File: spec/factories/browses.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
```
Comment (XaoGao) [inline spec/factories/browses.rb:1]: @sk8higher generate annotate

**Review by XaoGao — COMMENTED** [review]
@sk8higher pls fix

**Comment by sk8higher** [comment]
@XaoGao done

File: db/migrate/20231012080842_add_browses_count_to_vacancies.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class AddBrowsesCountToVacancies < ActiveRecord::Migration[7.0]
+  def change
+    add_column :vacancies, :browses_count, :integer
```
Comment (XaoGao) [inline db/migrate/20231012080842_add_browses_count_to_vacancies.rb:3]: default value 0, this filed also need to show in annotate in vacancy model class and factory

**Review by XaoGao — COMMENTED** [review]
@sk8higher pls fix

**Comment by sk8higher** [comment]
@XaoGao done

**Review by XaoGao — APPROVED** [review]
@sk8higher approve

**Pull request merged**